File: tests/indexing/parsing/lang/fixtures/php/Formatting.php
<?php

function tier(int $score): string
{
    $label = "";
    if ($score >= 90)
    {
        $label = "gold";
        $score -= 90;
    }
    elseif ($score >= 50)
    {
        $label = "silver";
    }
    else
    {
        $label = "bronze";
        $score = 0;
    }
    return $label;
}

function product(
    array $factors,
    int $start
): int {
    $total = 1;
    for (
        $i = $start;
        $i < count($factors);
        $i++
    ) {
        $total *= $factors[$i];
    }
    return $total;
}

File: tests/indexing/parsing/lang/fixtures/java/Formatting.java
class Formatting {

    String tier(int score)
    {
        String label;
        if (score >= 90)
        {
            label = "gold";
            score -= 90;
        }
        else if (score >= 50)
        {
            label = "silver";
        }
        else
        {
            label = "bronze";
            score = 0;
        }
        return label;
    }

    long product(
            int[] factors,
            int start
    ) {
        long total = 1;
        for (int i = start;
                i < factors.length;
                i++) {
            total *= factors[i];
        }
        return total;
    }

    int readValue(BufferedReader source, int fallback)
    {
        int value;
        try (
                BufferedReader reader = source;
                Closeable guard = source
        ) {
            value = Integer.parseInt(reader.readLine());
            value = value * 2;
        }
        catch (IOException
                | NumberFormatException e)
        {
            value = fallback;
        }
        return value;
    }
}
